### PR TITLE
Update keyboard plugin docs

### PR DESCRIPTION
**Pull request by KnorpelSenf**
Adds docs for https://github.com/grammyjs/grammY/pull/211

Please only merge once https://github.com/grammyjs/grammY/issues/211 is merged! This should not go live before 1.10.

### CODE REVIEW

File: site/docs/plugins/keyboard.md [hunk 1]
```diff
@@ -240,66 +240,67 @@ await ctx.reply(text, {
...
-    keyboard: keyboard.build(),
-  },
-});
+new Keyboard()
+  .text("Yes").row()
+  .text("No")
+  .oneTime();
 ```
 
+It does not matter when you call `oneTime` (first or last or somewhere in between).
```
Comment (roj1512) [inline site/docs/plugins/keyboard.md:273]:
```suggestion
It does not matter whether you call `oneTime` first, last or somewhere in between.
```

File: site/docs/plugins/keyboard.md [hunk 2]
```diff
@@ -240,66 +240,67 @@ await ctx.reply(text, {
...
-    keyboard: keyboard.build(),
-  },
-});
+new Keyboard()
+  .text("Yes").row()
+  .text("No")
+  .selected();
 ```
 
+It does not matter when you call `selected` (first or last or somewhere in between).
```
Comment (roj1512) [inline site/docs/plugins/keyboard.md:301]:
```suggestion
It does not matter whether you call `selected` first or last or somewhere in between.
```

File: site/docs/plugins/keyboard.md [hunk 3]
```diff
@@ -240,66 +240,67 @@ await ctx.reply(text, {
...
-    keyboard: keyboard.build(),
-  },
-});
+new Keyboard()
+  .text("Yes").row()
+  .text("No")
+  .placeholder("Decide now!");
 ```
 
+It does not matter when you call `placeholder` (first or last or somewhere in between).
```
Comment (roj1512) [inline site/docs/plugins/keyboard.md:287]:
```suggestion
It does not matter whether you call `placeholder` first, last or somewhere in between.
```

File: site/docs/plugins/keyboard.md [hunk 4]
```diff
@@ -240,66 +240,67 @@ await ctx.reply(text, {
...
-    keyboard: keyboard.build(),
-  },
-});
+new Keyboard()
+  .text("Yes").row()
+  .text("No")
+  .resized();
 ```
 
+It does not matter when you call `resized` (first or last or somewhere in between).
```
Comment (roj1512) [inline site/docs/plugins/keyboard.md:259]:
```suggestion
It does not matter whether you call `resized` first, last or somewhere in between.
```

Comment (KnorpelSenf) [reply site/docs/plugins/keyboard.md:259]: I like those a lot

**Review by roj1512 — APPROVED** [review]

File: site/docs/plugins/keyboard.md [hunk 5]
```diff
@@ -240,66 +240,67 @@ await ctx.reply(text, {
 
 Naturally, all other methods that send messages other than text messages support the same options, as specified by the [Telegram Bot API Reference](https://core.telegram.org/bots/api).
 
-If you want to specify further options with your message, you may need to create your own `reply_markup` object.
-In that case, you have to use `keyboard.build()` when passing it to your object.
+You can also give your keyboard one or more further properties by calling special methods on it.
+They will not add any buttons, but rather define the behavior of the keyboard:
```
Comment (habemuscode) [inline site/docs/plugins/keyboard.md:244]: `.` instead of `:`?

Comment (roj1512) [reply site/docs/plugins/keyboard.md:244]: Both are OK to me here, I didn't suggest it because it was a long sentence. @KnorpelSenf makes final decision.

**Review by habemuscode — APPROVED** [review]

File: site/docs/plugins/keyboard.md (hunk 5 above)
Comment (KnorpelSenf) [reply site/docs/plugins/keyboard.md:244]: Go ahead

Comment (habemuscode) [reply site/docs/plugins/keyboard.md]: [bb048cd](https://github.com/grammyjs/website/pull/451/commits/bb048cdbd65b979de7bd6c43357166fbea19f426)

**Pull request merged**